src/components/InventoryV2/Search.tsx

```diff
@@ -141,45 +187,83 @@ export default function Search(props: SearchProps): JSX.Element | null {
             onClickRightIcon={() => onSearch('')}
           />
         </Box>
-        <InventoryFilters filters={filters} setFilters={setFilters} origin={origin} />
+        {origin === 'Species' && (
```

Comment:
I thought it cleaner to move this logic out of the InventoryFilters component and into this one.